src/charmed_kubeflow_chisme/rock/_test_rock.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+from pathlib import Path
+
+import yaml
+
+
+class TestRock:
+    """Test class for ROCKs."""
+
+    def __init__(self):
+        """Initializes and return TestRock instance."""
+        # no initialization is currently needed
+
+    def get_rock_image_name_from_rockcraft(self, file):
+        """Read ROCK information and return ROCK image name."""
```

Comment:
Maybe:
```suggestion
    def get_rock_image_name_from_rockcraft(self, file: str):
        """Reads a rockcraft.yaml file and returns the ROCK image name."""
```
?

On first read, I wasn't sure what `file` should be.  Could also rename to `filename` so it is clear, but having type of `str` makes that pretty clear too